src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <h1>Calculator</h1>
+      <>
+        <Display />
+        <ButtonPanel />
+      </>
+    </div>
+  );
```

Comment:
Your use of [React.fragments](https://reactjs.org/docs/fragments.html#short-syntax) is kinda redundant since the fragment is expected to let you return a list of children without adding extra nodes to the DOM. In this case, it is expected to be the returned parent, not the `div`.